lib/spack/spack/solver/asp.py

```diff
@@ -115,6 +115,8 @@ class Provenance(enum.IntEnum):
     PACKAGE_PY = enum.auto()
     # An installed spec
     INSTALLED = enum.auto()
```

Comment:
I would like to rename this but it was causing a failure in the unit-tests that I could not uncover.  As far as I can tell it is only used in this file, but maybe there is something in the ASP that is sensitive and not hitting my grep searches? :shrug: